olp-cpp-sdk-core/src/http/curl/NetworkCurl.cpp

```diff
@@ -187,7 +187,8 @@ static curl_code SslctxFunction(CURL* curl, void* sslctx, void*) {
 
 }  // anonymous namespace
 
-NetworkCurl::NetworkCurl() {}
+NetworkCurl::NetworkCurl(size_t max_requests_count)
+    : handles_(max_requests_count) {}
```

Comment:
Please make sure that this does not explode if max_requests_count is smaller then kStaticHandleCount.

Reply:
Which scenario do you refer?

Reply:
Anyway, kStaticHandleCount now initialized as a quarter of max_requests_count.